packages/manager/apps/web/client/app/hosting/email/activate/activate.module.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+
+import routing from './activate.routing';
+import addComponent from './activate.component';
+
+const moduleName = 'webHostingEmailActivateModule';
+
+angular
+  .module(moduleName, [])
```

Comment:
No routing, translations or ui-kit ?

Reply:
translations are included, check line below
.run(/* @ngTranslationsInject:json ./translations */)

Reply:
But you need the angular translate module

Reply:
```suggestion
  .module(moduleName, [
   'oui',
   'pascalprecht.translate'
  ])
```
and all other modules you might need